app/assets/stylesheets/mobile/comments.scss

```diff
@@ -0,0 +1,66 @@
+.bottom_bar {
+  border-radius: 0 0 5px 5px;
+  z-index: 3;
+  display: block;
+  position: relative;
+  padding: 10px;
+  padding-top: 8px;
+  background: $background-grey;
+
+  margin: {
+    top: 10px; };
+
+  border: {
+    top: 1px solid $border-grey;
+  };
+
+  min-height: 22px;
+
+  > a,
+  .show_comments {
+    @include transition(color);
+    color: $text-grey;
+    font-weight: bold;
+  }
+
+  .show_comments {
+    position: relative;
+    top: 3px;
+    color: #ccc;
```

Comment:
Please use a color variable. Since the text is barely visible `$text-grey` could improve the visibility.